package/libs/zlib/Makefile

```diff
@@ -46,27 +49,21 @@ define Package/zlib-dev/description
  This package includes the development support files.
 endef
 
-CONFIGURE_VARS := \
```

Comment:
And we don't need `CONFIGURE_VARS` or `CONFIGURE_ARGS` either?

Reply:
All needed information is provided by the toolchain :-)